harry-runner/src/harry/runner/HarryRunner.java

```diff
@@ -36,21 +38,13 @@
 import harry.corruptor.QueryResponseCorruptor;
 import harry.model.ExhaustiveChecker;
 import harry.model.OpSelectors;
-import harry.model.sut.InJvmSut;
-import org.apache.cassandra.distributed.impl.IsolatedExecutor;
-import org.apache.cassandra.distributed.test.TestBaseImpl;
 
-public class HarryRunner extends TestBaseImpl
+public interface HarryRunner
```

Comment:
I turned this into an interface with a default implementation that's further implemented by both `HarryRunnerJvm` and `HarryRunnerExternal` and removed the dependency on `TestBaseImpl`.

Reply:
In this case, should we move inteface to `core`?

Reply:
Done!
And removed the harry-runner module completely.